core/import-seac.cpp

```diff
@@ -123,7 +122,9 @@ static int seac_dive(void *param, int, char **data, char **)
 	 "-13:45",  // 40
 	 "-14:00"}; // 41
 
-	sprintf(isodatetime, "%4i-%02i-%02iT%02i:%02i:%02i%6s", year, month, day, hour, min, sec, timezoneoffset[tz]);
+	constexpr int BUFSIZE = 30;
```

Comment:
I suggest replacing these by 
```
        std::string isodatetime = format_string_std(...);
```
and then access the C string by `isodatetime.c_str()`.

It's not like this is performance sensitive...